Tests/Core/Networking/RequestTests.swift

```diff
@@ -0,0 +1,218 @@
+import XCTest
+
+@testable import Frames
+
+final class RequestTests: XCTestCase {
```

Comment:
would it make sense to parameterise some of the tests here

Reply:
I think that the tests here are small and well-defined, but I'd be more than happy to consider any suggestions 😄

Reply:
You could definitely group these tests. I would do so by method on request. The different arguments and results to the methods on `Request` should be simple to parameterise.